package.json

```diff
@@ -22,10 +22,12 @@
   "engines": {
     "node": "8.9.4"
   },
-  "main": "src/index.js",
+  "main": "vast-client-node.js",
```

Comment:
This file is not generated in the repo, you forgot to `npm run build` and check it in :)

Reply:
We don't commit the bundled versions, only for the release https://github.com/dailymotion/vast-client-js/blob/master/docs/CONTRIBUTING.md#pull-requests. We can definitely make an exception here :p

Reply:
ah fine 👍 let's do it before 2.0.1 ;)